.github/workflows/lint.yml

```diff
@@ -1,50 +1,54 @@
-name: Linters
+# lint runs all linters in this repository. This workflow is run on every pull
+# request and push to main.
+name: lint
 on:
   push:
     tags:
       - v*
     branches:
       - main
   pull_request:
+  workflow_call:
 
 jobs:
-  golangci:
-    name: lint
+  golangci-lint:
+    name: golangci-lint
     runs-on: ubuntu-latest
+    timeout-minutes: 8
```

Comment:
[question] have there been instances of this lint hanging? just curious as the rationale for adding the timeout.

Reply:
![meme](https://user-images.githubusercontent.com/1138893/229856985-9da119ad-a8da-43b7-a7a4-3f5db08b7046.png)

Reply:
![meme](https://user-images.githubusercontent.com/1138893/229859589-849fa5b1-70c8-4a99-95ae-0dd3e093ba47.png)

Reply:
Removed this in favor of the timeout in `golangci/golangci-lint-action`. Other steps should not hang and if the timeout of the lint actions hit the job can terminate without a job timeout.